tests/test_wps_mmdmt.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from tempfile import NamedTemporaryFile
+
+from wps_tools.testing import run_wps_process, local_path
+from quail.processes.wps_climdex_mmdmt import ClimdexMMDMT
+
+
+@pytest.mark.parametrize(
+    ("climdex_input", "ci_name", "month_type", "freq"),
+    [
+        (local_path("climdexInput.rda"), "ci", "txx", "monthly"),
+        (local_path("climdexInput.rda"), "ci", "txx", "annual"),
+    ],
+)
```

Comment:
As mentioned above, it would be nice to see test cases dealing with multiple `freq` values.